Youtube-dl GUI.py

```diff
@@ -82,12 +81,12 @@ def check_for_update(a,b=1):
     file7.write("%(playlist_title)s %(playlist_index)s %(title)s.%(ext)s")
     file7.close()
     
-t1xz = threading.Thread(target=check_for_update, args=("https://sourabhkv.github.io/ytdl",))

```

Comment:
Next time we will use GitHub API

Reply:
https://api.github.com/repos/sourabhkv/ytdl/releases/latest

Reply:
This wont fetch us tags of previous releases
use this https://api.github.com/repos/{owner}/{repo}/tags